src/components/MediaSettings/MediaSettingsTabs.vue

```diff
@@ -0,0 +1,139 @@
+<!--
+  - SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<script setup lang="ts">
+import { computed, ref } from 'vue'
+import type { Component } from 'vue'
+
+import NcButton from '@nextcloud/vue/dist/Components/NcButton.js'
+
+import TransitionExpandDown from './TransitionExpandDown.vue'
+
+type TabDefinition = {
+	id: string,
+	label: string,
+	icon: Component,
+}
+
+const props = defineProps<{
+    tabs: TabDefinition[],
+    active?: string,
+}>()
+
+const emit = defineEmits<{
+    (event: 'update:active', value?: string): void
+}>()
+
+/** Whether the tab panel is open */
+const isOpen = ref(false)
```

Comment:
That solves auto-opening, please check it doesn't break anything else =)
```suggestion
const isOpen = ref(!!props.active)
```

Reply:
Added support for open state change from parent via falsy `active` (2-ways)